src/components/ShowMoreScholarships.jsx

```diff
@@ -0,0 +1,59 @@
+import { useContext, useRef, useEffect } from 'react';
+import ScholarshipsContext from '../models/ScholarshipsContext';
+import ShowMoreScholarshipCard from './ShowMoreScholarshipCard';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import useOnScreen from '../lib/useOnScreen';
+import { useTranslation } from 'react-i18next';
+
+export default function ShowMoreScholarships() {
+  const { canLoadMore, error, loading, loadMore, scholarships, setFilters } =
+    useContext(ScholarshipsContext);
+  const { t } = useTranslation();
+
+  console.log(scholarships);
+
+  console.log(t);
+  let s = scholarships.map((scholarship, id) => ({ [id]: scholarship.data }));
+  console.log(s);
+  let h = scholarships.map((scholarship, id) => ({
+    [id]: scholarship.data.name,
+  }));
+  console.log(h);
+  let k = scholarships.map((scholarship, id) => ({
+    [id]: scholarship.data.amount.min,
+  }));
+  console.log(k);
+  // Automatically load more when the progress is visible
+  const progressRef = useRef(null);
+  const progressVisible = useOnScreen(progressRef);
+
+  useEffect(() => {
+    if (progressVisible && canLoadMore && !loading) {
+      loadMore();
+    }
+  }, [loading, canLoadMore, loadMore, progressVisible]);
+
+  const j = canLoadMore;
+  console.log(j);
+  const e = error;
+  console.log(e);
+  console.log(loading);
+  console.log(loadMore);
+  console.log(setFilters);
+  console.log(scholarships);
+
+  return (
+    <Box>
+      <Grid container spacing={2} justify="center">
+        {scholarships.slice(0, 3).map((scholarship, id) => {
+          return (
+            <Grid key={id} item xs={12} sm={6} md={4}>
+              <ShowMoreScholarshipCard scholarship={scholarship} />
```

Comment:
You could probably reuse **ScholarshipCard** and use it like ListScholarships does instead since it already displays the basic info:
https://github.com/beyondhb1079/s4us/blob/3b4eac7611b282bc5d2e3e095a3a15a116e2d9cb/src/components/ScholarshipList.jsx#L29-L31